src/Shared/UnitTests/AssemblyNameEx_Tests.cs

```diff
@@ -676,6 +679,69 @@ public void VerifyReverseVersionComparer()
             Assert.True(assemblies[1].Equals(z));
             Assert.True(assemblies[2].Equals(y));
         }
+
+        [Theory]
+        [InlineData("System.Xml")]
+        [InlineData("System.XML, Version=2.0.0.0")]
+        [InlineData("System.Xml, Culture=de-DE")]
+        [InlineData("System.Xml, Version=10.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, Retargetable=Yes")]
+        [InlineData("System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
```

Comment:
Which of these would have failed before?

Reply:
That's on my list to do :)